test/unit/url.cpp

```diff
@@ -288,6 +288,12 @@ struct url_test
             BOOST_TEST( ! u.is_path_absolute() );
             BOOST_TEST_EQ(u.encoded_path(), "");
         }
+        {
+            // issue 390
+            url u( "/kyle:xy" );
+            u.set_path_absolute( false );
+            BOOST_TEST_EQ( u.buffer(), "./kyle:xy" );
```

Comment:
I like breaking out individual tests for issues only after the initial release. Before then ,we can do what we want. But I will not hold up this PR for that.